app/build.gradle

```diff
@@ -2,11 +2,12 @@ apply plugin: 'com.android.application'
 apply plugin: 'kotlin-android'
 
 android {
-	namespace 'de.markusfisch.android.binaryeye'
-	compileSdkVersion sdk_version
 	defaultConfig {
-		minSdkVersion 9
```

Comment:
Because the actual value, "9", is used just once, in this place, I don't see much value in defining a global symbol for it.

Reply:
This seems reasonable as well, I'm reverting the change.